formats/finder_email_signup/publisher/details.json

```diff
@@ -9,7 +9,7 @@
   ],
   "properties": {
     "beta": {
-      "type": "boolean"
+      "$ref": "#/definitions/finder_beta"
```

Comment:
Is it going to be a problem that this now allows null, when before it had to be true or false?  Is it worth checking the rendering app for this - if it uses the same finder frontend it's probably fine.